Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/MenuPage.xaml

```diff
@@ -0,0 +1,414 @@
+﻿<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.MenuPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:commands="using:Microsoft.Toolkit.Uwp.SampleApp.Menu.Commands"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <commands:NewProjectCommand x:Key="NewProject" />
+        <commands:NewFileCommand x:Key="NewFile" />
+        <commands:GenericCommand x:Key="GenericCommand" />
+        <Style TargetType="controls:MenuItem">
+            <Setter Property="Background" Value="{StaticResource Brush-Grey-01}" />
```

Comment:
Remove the background from here and add to the menu so the menu has a consistent background across the top